tutorials/epochs/15_baseline_regression.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
+"""
+.. _ex-baseline-regression:
+
+====================================================================
+Regression-based baseline correction
+====================================================================
+
+This tutorial compares traditional baseline correction (adding or subtracting a
+scalar amount from every timepoint in an epoch) to a regression-based approach
+to baseline correction (which allows the effect of the baseline period to vary
+by timepoint). Specifically, this tutorial follows the method introduced by
+:footcite:t:`Alday2019`.
+
+There are at least two reasons you might consider using regression-based
+baseline correction:
+
+1. Unlike traditional baseline correction, the regression-based approach does
+   not assume that the effect of the baseline is equivalent between different
+   experimental conditions. Thus it is safer against introduced bias /
+   false-positives.
```

Comment:
```suggestion
   experimental conditions. Thus it is safer against introduced bias.
```

OR

```suggestion
   experimental conditions. Thus it is safer against introduced bias
   from carrying over differences in the baseline interval into the
   window of interest.
```

I would leave out the stronger statistical commentary here, because it requires a bit of subtlety and qualification. I did find a probable false positive in the example in the paper, but it's hard to know the ground truth of this.